guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -18,14 +18,84 @@ You can list the available ece-tools commands using:
 php ./vendor/bin/ece-tools list
 ```
 
-The following updates describe the latest improvements to the ece-tools package, which update as needed through patching and product upgrades managed by the `magento-cloud-metapackage`. The ece-tools package adheres to the version sequence:  `200<major>.<minor>.<patch>`.
+The following updates describe the latest improvements to the `ece-tools` package, which uses the following version sequence:  `200<major>.<minor>.<patch>`.  See [Upgrades and patches]({{ site.baseurl }}/guides/v2.1/cloud/project/project-upgrade-parent.html) for information about updating to the latest release of the `ece-tools` package.
+
+
+## v2002.0.13
```

Comment:
MAGECLOUD-2464 is missing from release notes. Disregard the placement of this comment, just need to add it somewhere.

Reply:
Added: 

MAGECLOUD-2464:  Fixed a compatibility issue in the `ece-tools` package that caused problems with cron queue management in Magento versions 2.2 - 2.5.